modules/support_ticket/config/install/views.view.users_support_tickets.yml

```diff
@@ -340,13 +335,10 @@ display:
           delta_first_last: false
           multi_type: separator
           separator: ', '
-          field_api_classes: 0
-          date_format: 'time ago'
-          custom_date_format: ''
-          timezone: ''
+          field_api_classes: false
           entity_type: support_ticket
           entity_field: changed
-          plugin_id: date
+          plugin_id: field
```

Comment:
I'm wondering whether instead adding 'type: timestamp' would be better, but I guess it will pick the default formatter automatically for you